backend/models/data.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Column, Integer, String, DateTime, ForeignKey
+from sqlalchemy.sql import func
+from database import Base
+import models.file
+
+
+class Data(Base):
+    __tablename__ = "data"
+    file_id = Column(Integer, ForeignKey("file.id"), primary_key=True)
```

Comment:
This table could also have id as primary key.

Reply:
Solved and I added `UNIQUE` constraint for `(file_id, keyword)` to let each file cannot has any duplicate keywords